src/dist/nfs/nfs_client_impl.cpp

```diff
@@ -40,7 +40,57 @@
 namespace dsn {
 namespace service {
 
+DSN_DEFINE_uint32("nfs",
+                  nfs_copy_block_bytes,
+                  4 * 1024 * 1024,
+                  "max block size (bytes) for each network copy");
 DSN_DEFINE_int32("nfs", max_copy_rate_megabytes, 500, "max rate of copying from remote node(MB/s)");
+DSN_DEFINE_int32("nfs",
+                 max_concurrent_remote_copy_requests,
+                 50,
+                 "max concurrent remote copy to the same server on nfs client");
+DSN_DEFINE_int32("nfs", max_concurrent_local_writes, 50, "max local file writes on nfs client");
+DSN_DEFINE_int32("nfs", max_buffered_local_writes, 500, "max buffered file writes on nfs client");
+DSN_DEFINE_int32("nfs",
+                 high_priority_speed_rate,
+                 2,
+                 "the copy speed rate of high priority comparing with low priority on nfs client");
+DSN_DEFINE_int32("nfs",
+                 file_close_expire_time_ms,
+                 60 * 1000,
+                 "max idle time for an opening file on nfs server");
+DSN_DEFINE_int32("nfs",
+                 file_close_timer_interval_ms_on_server,
+                 30 * 1000,
+                 "time interval for checking whether cached file handles need to be closed");
+DSN_DEFINE_int32("nfs",
+                 max_file_copy_request_count_per_file,
+                 2,
+                 "maximum concurrent remote copy requests for the same file on nfs client"
+                 "to limit each file copy speed");
+DSN_DEFINE_int32("nfs",
+                 max_retry_count_per_copy_request,
+                 2,
+                 "maximum retry count when copy failed");
+DSN_DEFINE_int32("nfs",
+                 rpc_timeout_ms,
+                 10000,
+                 "rpc timeout in milliseconds for nfs copy, "
+                 "0 means use default timeout of rpc engine");
+
+void nfs_opts::init()
+{
+    nfs_copy_block_bytes = FLAGS_nfs_copy_block_bytes;
```

Comment:
You can use FLAGS_nfs_copy_block_bytes directly, nfs_copy_block_bytes can be removed.
Another profit of using FLAGS_xxx is to support dynamic configs modification in the future.